src/main/java/com/books/services/AuthenticateService.java

```diff
@@ -0,0 +1,26 @@
+package com.books.services;
+
+import com.books.entities.User;
+import com.books.storage.abstracts.UserDAO;
+import com.books.storage.concrete.SQL.UserRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AuthenticateService {
+    UserDAO repository;
```

Comment:
You forgot the priority of defining class elements I told you in first code review. Please, follow these rules :)